sedai/README.md

```diff
@@ -0,0 +1,47 @@
+# [Sedai][1] — Find out why production is the new playground with a free 14 day trial
+## Continuous Autonomous Availability Management (CAAM)
```

Comment:
We'll have to remove this level 2 heading as well

Reply:
done